### PR TITLE
Borgs can activate the emergency tesla beacon

### LINKED ISSUE #13357: Stationbounds Can't Activate or Deactivate the Emergency Tesla Beacon
title.

### PR DESCRIPTION
**Pull request by Doxxmedearly**
Fixes  #13357

### CODE REVIEW

**Comment by Doxxmedearly** [comment]
!review

File: code/game/machinery/syndicatebeacon.dm [hunk 1]
```diff
@@ -117,20 +117,17 @@
 	if(user)
 		to_chat(user, "<span class='notice'>You deactivate the beacon.</span>")
 
-
-/obj/machinery/power/singularity_beacon/attack_ai(mob/user as mob)
+/obj/machinery/power/singularity_beacon/emergency/attack_ai(mob/user)
 	return
```
Comment (NonQueueingMatt) [inline code/game/machinery/syndicatebeacon.dm:121]: Why is this here if it just gets overridden below?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/game/machinery/syndicatebeacon.dm (hunk 1 above)
Comment (Doxxmedearly) [reply code/game/machinery/syndicatebeacon.dm:121]: whoops, this should not be on the beacon subtype

**Review by NonQueueingMatt — APPROVED** [review]
